ownCloud/Settings/MediaUploadSettingsSection.swift

```diff
@@ -17,14 +17,24 @@
 */
 
 import UIKit
+import Photos
+import ownCloudSDK
 
 extension UserDefaults {
 
 	enum MediaUploadKeys : String {
 		case ConvertHEICtoJPEGKey = "convert-heic-to-jpeg"
 		case ConvertVideosToMP4Key = "convert-videos-to-mp4"
+		case InstantUploadPhotosKey = "instant-upload-photos"
+		case InstantUploadVideosKey = "instant-upload-videos"
+		case InstantUploadBookmarkUUIDKey = "instant-upload-bookmark-uuid"
+		case InstantUploadPathKey = "instant-upload-path"
+		case InstantUploadPhotosAfterDateKey = "instant-upload-photos-after-date"
+		case InstatnUploadVideosAfterDateKey = "instant-upload-videos-after-date"
```

Comment:
This should be `InstantUploadVideosAfterDateKey`.

Reply:
@felix-schwarz fixed